ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- assisted-cicd
```

Comment:
Can [now](https://github.com/openshift/release/pull/21322) use the `single-node` group thanks to @osherdp